cassandra/policies.py

```diff
@@ -376,7 +380,22 @@ def make_query_plan(self, working_keyspace=None, query=None):
                 for host in child.make_query_plan(keyspace, query):
                     yield host
             else:
-                replicas = self._cluster_metadata.get_replicas(keyspace, routing_key)
+                if self._experimental_tablets_enabled and self._cluster_metadata._tablets is not None:
+                    replicas = self._cluster_metadata._tablets.get_replicas_for_key(keyspace, query.table, self._cluster_metadata.token_map.token_class.from_key(routing_key))
```

Comment:
This name needs to be different, it's confusing because you define `replicas` again few lines later.

Reply:
Renamed